load_tests/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.7
+
+EXPOSE 8089
+WORKDIR /app
+
+# install poetry
+
+ARG POETRY_VERSION="1.0.0"
+RUN \
+  pip install --upgrade pip \
+  && pip install "poetry==${POETRY_VERSION}" \
+  && poetry config virtualenvs.create false
+
+COPY load_tests .
```

Comment:
Why naming it `load_tests` when everything else uses `-`?

Reply:
Renamed